sys/src/cmd/cifs/auth.c

```diff
@@ -63,7 +47,7 @@ auth_plain(char *windom, char *keyp, uint8_t *chal, int len)
 		sysfatal("cannot get key - %r");
 
 	ap = emalloc9p(sizeof(Auth));
-	memset(ap, 0, sizeof(ap));
+	memset(&ap->mackey, 0, sizeof(ap->mackey));
```

Comment:
Why did this change?  Compiler warning?

Reply:
deanon structs. Compiler tells you what it wants.